test/test_images/logevents/main.go

```diff
@@ -23,7 +23,8 @@ import (
 )
 
 func handler(event cloudevents.Event) {
-	log.Printf("%s", event)
+	// NOTE(Chi): Comment out since we want to count the number of event messages in the log, and the following code also prints it out
```

Comment:
It seems unnecessary to comment about disabling a debugging command. I suggest you simply remove both lines.